_dev/src/ts/modals/StartUpdateModal.ts

```diff
@@ -0,0 +1,70 @@
+import ComponentAbstract from '../types/ComponentAbstract';
+import api from '../api/RequestHandler';
+
+export default class StartUpdateModal extends ComponentAbstract {
+  protected readonly formId = 'form-confirm-update';
+  protected readonly confirmCheckboxId = 'modal-start-update-own-backup';
+
+  public mount(): void {
+    this.form.addEventListener('submit', this.onSubmit);
+    this.form.addEventListener('change', this.onChange);
+
+    this.updateSubmitButtonStatus(
+      document.getElementById('modal-start-update-own-backup') as HTMLInputElement | undefined
+    );
+  }
+
+  public beforeDestroy(): void {
+    this.form.removeEventListener('submit', this.onSubmit);
+    this.form.removeEventListener('change', this.onChange);
+  }
+
+  private get form(): HTMLFormElement {
+    const form = document.forms.namedItem('form-confirm-update');
+    if (!form) {
+      throw new Error('Form not found');
+    }
+
+    ['routeToSubmit'].forEach((data) => {
```

Comment:
```suggestion
    // We implement the same way to check from the other scripts, even though there is only one value.
    // This will ease any potential refacto.
    ['routeToSubmit'].forEach((data) => {
```